docs/assets/css/main.css

```diff
@@ -18720,3 +18722,34 @@ footer {
     background: rgba(5, 165, 146, 0.3);
     margin-top: 10px;
   }
+
+/*image-thumbnail-grid*/
+  
+.row {
+    display: flex;
+  }
+  
+.column {
+    flex: 25.00%;
+    padding: 5px;
+  }
+
+.img-thumbnail {
+    border: 1px solid #ddd;
```

Comment:
Remove border or if you'd like to make these buttons, reuse the existing button style (or set a new one) on the site.